File: experiment_jp_test/mutants/for__0__FREQUENT_ELEMENT_ARRAY_1.java
static int mostFrequent ( int arr [ ] , int n ) { Map < Integer , Integer > hp = new HashMap < Integer , Integer > ( ) ; int max_count = 0 , res = - 1 ; for ( Entry < Integer , Integer > val : hp . entrySet ( ) ) { if ( max_count < val . getValue ( ) ) { res = val . getKey ( ) ; max_count = val . getValue ( ) ; } } return res ; }

File: experiment_jp_test/j2pyrawpy/for__0__FREQUENT_ELEMENT_ARRAY_1.py
#!/usr/bin/env python
""" generated source for module for__0__FREQUENT_ELEMENT_ARRAY_1 """
class X(object):
    """ generated source for class X """
    @classmethod
    def mostFrequent(cls, arr, n):
        """ generated source for method mostFrequent """
        hp = HashMap()
        max_count = 0
        res = -1
        for val in hp.entrySet():
            if max_count < val.getValue():
                res = val.getKey()
                max_count = val.getValue()
        return res

